spec/imsc-hrm.html

```diff
@@ -517,16 +524,18 @@ <h2>General</h2>
         </li>
       </ul>
 
-      <p>The contents of the Presentation Buffer P<sub>n</sub> SHALL be transferred instantaneously to Presentation Buffer
-      P<sub>n-1</sub> at the presentation time of <a>Intermediate Synchronic Document</a> E<sub>n</sub>, making the latter
-      available for display.</p>
+      <p>The contents of the Front Buffer SHALL be transferred instantaneously to the Back Buffer at the presentation
+      time of a <a>non-empty ISD</a> E<sub>n</sub>, making the latter available for display.</p>
+
+      <p>The Back Buffer SHALL be disconnected from the display while <a>empty ISD</a> E<sub>n</sub> is being
+      presented.</p>
 
-      <p class='note'>It is possible for the contents of Presentation Buffer P<sub>n-1</sub> to never be displayed. This can happen
-      if Presentation Buffer P<sub>n</sub> is copied twice to Presentation Buffer P<sub>n-1</sub> between two consecutive video
-      frame boundaries of the <a>Related Video Object</a>.</p>
+      <p class='note'>It is possible for the contents of the Back Buffer to never be displayed. This can happen, for
+      example, if the Front Buffer is copied twice to Back Buffer between two consecutive video frame boundaries of the
+      <a>Related Video Object</a>.</p>
 
-      <p>It SHALL be an <a>error</a> for the Presentation Compositor to fail to complete painting pixels for E<sub>n</sub> before the
-      presentation time of E<sub>n</sub>.</p>
+      <p>It SHALL be an <a>error</a> for the Presentation Compositor to fail to complete painting pixels for
+      <a>non-empty ISD</a> E<sub>n</sub> before the presentation time of <a>non-empty ISD</a> E<sub>n</sub>.</p>
```

Comment:
Again, shouldn't the subscripts be different from each other here, e.g. use _n_ and _m_?

Reply:
The error occurs if the painting a non-empty ISD does not complete before the presentation time of that ISD. Repeating _non-empty ISD E_n_ makes the sentence confusing.

Reply:
See https://github.com/w3c/imsc-hrm/pull/50/commits/aba90ba175e83f6cd684c91b854cf27d49169f26

Reply:
Much clearer now, thank you.